account_analytic_billing_plan/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Account Analytic Billing Plan",
+    "version": "12.0.1.0.0",
```

Comment:
@oihane If added new Module then the version should be `"12.0.0.0.0"`.
Or Add `Summary` key

Reply:
That's not true... Version must be always 12.0.1.0.0 for new version, and summary is not mandatory.